bech32/app/Main.hs

```diff
@@ -48,15 +50,22 @@ import qualified Codec.Binary.Bech32 as Bech32
 import qualified Codec.Binary.Bech32.Internal as Bech32
 import qualified Data.ByteArray.Encoding as BA
 import qualified Data.ByteString.Char8 as B8
+import qualified Data.List as L
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as T
+import qualified Options.Applicative as OA
```

Comment:
The coding style of this module uses less qualified imports.
I don't think there need to be any more qualified imports than what there currently are.
Even `import Paths_bech32 (version)` is fine.